t/Local.t

```diff
@@ -399,4 +372,78 @@ subtest(
     },
```

Comment:
Not sure if it's necessary but I added a 'posix' version of this subtest to ensure interpretation is consistent.

Reply:
I think it's a good idea. I also changed most of the test code to call all of the relevant subs for each test except for parts that are specific to just one implementation.